extension/content.css

```diff
@@ -1,16 +1,15 @@
-mark {
-}
 .markedYellow {
-	background-color:yellow!important;
+	background-color: yellow;
```

Comment:
Need to remove !important, or else the fadeaway style will always get overwritten.

Reply:
LGTM

Reply:
LGTM